docs/docs/91-migrations.md

```diff
@@ -20,6 +20,7 @@ Some versions need some changes to the server configuration or the pipeline conf
 - Removed `WOODPECKER_DOCS` config variable
 - Renamed `link` to `url` (including all API fields)
 - Deprecated `CI_COMMIT_URL` env var, use `CI_PIPELINE_FORGE_URL`
+- Migrated from Bitbucket Stash support to Bitbucket Datacenter / server support, rename env `WOODPECKER_STASH` to `WOODPECKER_BITBUCKET_DC` and webhooks need to be re-installed (e.g., using "repair" on the project in Woodpecker)
```

Comment:
We removed it in the past ... but thats long gone. So a migration should be not mentioned

Reply:
Sure we can delete it - it was merely because this PR originally referenced #2364 where someone was running the older implementation of BB support.